locale/fi/notification.json

```diff
@@ -0,0 +1,59 @@
+{
+  "title": "Ilmoitus (Notification)",
+  "intro": "Ilmoitus on komponentti, jonka avulla käyttäjälle voidaan välittää olennaista palvelun osaan ja käyttäjän toimintaan liittyvää palautetta ja tietoa. ",
+  "note.title": "Saavutettavuus ja käytettävyys",
+  "note.items": [
+    "Mikäli ilmoituksia on tarkoitus lisätä sivulle dynaamisesti, niitä varten kannattaa luoda ruudunlukijakäyttäjiä varten live region, jonka sisään ilmoitukset lisätään. Tällöin myös ruudunlukijakäyttäjät saavat tiedon sivulle lisätystä ilmoituksesta.",
+    "Ilmoituksen otsikkotaso on määriteltävissä käyttötapauksen mukaan. Käytä ilmoituksessa otsikkotasoa, joka ei riko näkymän otsikkojärjestystä.",
+    "Sivulla voi olla yhtä aikaa useita ilmoituksia näkyvissä.",
+    ""
+  ],
+  "sections": [
+    {
+      "title": "Mistä komponentti koostuu",
+      "paragraphs": [
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Komponentti koostuu ilmoituksen tyyppiä kuvaavasta ikonista, otsikosta ja pääsisällöstä sekä \"sulje\" painikkeesta. Ilmoituksen pääsisällön alle on myös mahdollista lisätä toimintoelementtejä, kuten painikkeita tarpeen vaatiessa."
```

Comment:
> myös mahdollista lisätä toimintoelementtejä, kuten painikkeita, tarpeen vaatiessa.

I would add one comma (after the "**painikkeita**") for this to be easier for reader and maybe more grammatically correct (?)

Reply:
Removed the "tarpeen vaatiessa" from the end, since it was mostly redundant.